lib/lichess.py

```diff
@@ -253,21 +253,23 @@ def rate_limit_time_left(self, path_template: str) -> datetime.timedelta:
         """How much time is left until we can use the path template normally."""
         return self.rate_limit_timers[path_template].time_until_expiration()
 
-    def upgrade_to_bot_account(self) -> JSON_REPLY_TYPE:
+    def upgrade_to_bot_account(self) -> None:
         """Upgrade the account to a BOT account."""
-        return self.api_post("upgrade")
+        self.api_post("upgrade")
 
-    def make_move(self, game_id: str, move: chess.engine.PlayResult) -> JSON_REPLY_TYPE:
+    def make_move(self, game_id: str, move: chess.engine.PlayResult) -> None:
         """
         Make a move.
 
         :param game_id: The id of the game.
         :param move: The move to make.
         """
-        return self.api_post("move", game_id, move.move,
-                             params={"offeringDraw": str(move.draw_offered).lower()})
+        self.api_post(
+            "move", game_id, move.move,
+            params={"offeringDraw": str(move.draw_offered).lower()}
+        )
```

Comment:
To keep the style consistent: (1) put the first line of arguments on the same line as the function name, (2) put the final closing parenthesis on the same line as the last arguments, and (3) line up the arguments with the opening parenthesis of the function.
```python
        self.api_post("move", game_id, move.move,
                      params={"offeringDraw": str(move.draw_offered).lower()})
```

Reply:
Sounds good.

Btw any interest in black or ruff for this repo? They can handle this sort of formatting stuff automatically.

Reply:
We've considered it, but we prefer the flexibility to arrange code that makes sense in context and using flake8 to keep things tidy. I just tried running black on this repository and there's maybe 1 in 10 changes I agree with. Personally, I don't like how black puts the closing parenthesis/brace on a new line in a multi-line list, nor putting a comma after the last element of a list.